02LessonTwo/homework/index.html

```diff
@@ -16,13 +16,12 @@
 
```

Comment:
you've got all the content needed and the page is looking good on the screen! however, your code needs a little cleaning up. remember to indent whenever there is a parent>child relationship (also called nesting). for example, line 26 has an h3 and line 28 has a p. the p is not the child of the h3, it is the child of the article. siblings will be on the same vertical line.

Also, please start getting into the habit of commenting your code :)  it will help later!

Great start Manny!!